assets/src/scripts/monitoring-location/components/hydrograph/date-controls.js

```diff
@@ -102,20 +133,22 @@ export const drawDateRangeControls = function(elem, store, siteno) {
         .attr('id', 'custom-date-submit')
         .text('Submit')
         .on('click', function() {
-            const userSpecifiedStart = customStartDate.node().value;
-            const userSpecifiedEnd = customEndDate.node().value;
+            let userSpecifiedStart = document.getElementById('custom-start-date').value
+            let userSpecifiedEnd = document.getElementById('custom-end-date').value
             if (userSpecifiedStart.length === 0 || userSpecifiedEnd.length === 0) {
                 dateAlertBody.selectAll('p').remove();
                 dateAlertBody.append('p')
                     .text('Both start and end dates must be specified.');
                 customDateValidationContainer.attr('hidden', null);
-            } else if (DateTime.fromISO(userSpecifiedEnd) < DateTime.fromISO(userSpecifiedStart)) {
+            } else if (DateTime.fromFormat(userSpecifiedEnd, 'LL/dd/yyyy') < DateTime.fromFormat(userSpecifiedStart, 'LL/dd/yyyy')) {
```

Comment:
The way the USWDS component works, the display format is the actual value with no abstraction layer. So I had to operate with this format and compel the dates back to ISO for pushing to the redux store (see lines 150-152 below)